src/OpenLoco/src/World/CompanyAi.cpp

```diff
@@ -4238,61 +4240,634 @@ namespace OpenLoco
         aiThinkEndCompany,
     };
 
-    // 0x00431295
-    static void sub_431295(Company& company)
+    namespace PlaceVehicleStateMachine
```

Comment:
I've put all of this statemachines code in a namespace I'm thinking of moving it into a separate source file.

Reply:
I think that would be good as well. This file has gotten very long as-is. Maybe the company AI unit should be split up into several smaller units, anyway, and live in its own folder? Definitely not for this PR, of course.